homeassistant/components/smartthings/lock.py

```diff
@@ -63,7 +62,7 @@ async def async_unlock(self, **kwargs: Any) -> None:
     @property
     def is_locked(self) -> bool:
         """Return true if lock is locked."""
-        return self._device.status.lock == ST_STATE_LOCKED
+        return self._device.status.lock == LockState.LOCKED
```

Comment:
I think this should be reverted since it compares the smart things device state.

Reply:
Yes. No idea how I managed to change that. Will revert it.

Reply:
Has been addressed and I went through all the updates and could not find any more incorrect occurrences of things that shouldn't change.